Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -618,6 +618,8 @@ YUI.add('ez-platformuiapp', function (Y) {
             });
             if ( createView ) {
                 viewInfo.instance = new viewInfo.type();
+            } else {
+                viewInfo.instance.reset();
```

Comment:
this change affects each and every side view, did you check what happen with others side views ?